pysrvx/srvx.py

```diff
@@ -125,8 +125,20 @@ def generate_token(self):
         """Return a token generated from a random number"""
         return 'GS%05d' % randint(0,65535)
 
-    def get_response(self):
+    @staticmethod
+    def decode_string(tmp):
+        decoded_string = ''
```

Comment:
useless, you always assign something later - also, you can just `return` in each of the next three blocks